digital_image_processing/change_contrast.py

```diff
@@ -0,0 +1,35 @@
+"""
+Changing contrast with PIL
+
+This algorithm is used in
+https://noivce.pythonanywhere.com/ python web app.
+
+python/black: True
+flake8 : True
+"""
+
+from PIL import Image
+
+
+def change_contrast(img, level: int):

```

Comment:
def change_contrast(img: Image, level: int) -> ???:

Reply:
On it.